osu.Game/Overlays/Profile/Sections/Ranks/DrawableProfileScore.cs

```diff
@@ -268,21 +269,23 @@ private Drawable createDrawablePerformance()
                 Direction = FillDirection.Horizontal,
                 Children = new[]
                 {
-                    new OsuSpriteText
+                    new SpriteTextWithTooltip
                     {
                         Anchor = Anchor.BottomLeft,
                         Origin = Anchor.BottomLeft,
                         Font = font,
-                        Text = $"{Score.PP:0}",
-                        Colour = colourProvider.Highlight1
+                        Text = Score.PP.ToLocalisableString(@"N0"),
+                        TooltipText = Score.PP.ToLocalisableString(@"0.###"),
```

Comment:
now there won't be any thousands separators in the tooltips

Reply:
o_o

Reply:
I'm not sure we should *ever* show 3 decimals. I'd rather set a good example and not show more than *one at maximum*, then have osu-web match.